### DISCUSSION TITLE
"Please check your dataset directories"

### DISCUSSION THREAD

**Discussion by KevMolya** [d8ahazard/sd_dreambooth_extension#1164] [Q&A]
I installed Dreamboot yesterday and I watch many tutos to try to fix my problem but I can't. I always get "Please check your dataset directories" when I hit TRAIN. I read elsewhere to fix Class Image Generation Class Images Per Instance Image but it's not working.
Can someone help me?

**Comment by Stoascheisserkoarl** [comment]
Same thing to me

**Comment by plasticarm** [comment]
I ran into the same problem. The answer was to restart the UI.
